src/App.js

```diff
@@ -1,12 +1,161 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Mainbutton from "./components/Mainbutton";
+import Input from "./components/Input";
+import Fruitcounter from "./components/Fruitcounter";
+import emoji from "emoji-dictionary";
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+    const [strawberryCounter, setStrawberryCounter] = useState(0);
+    const [bananaCounter, setBananaCounter] = useState(0);
+    const [appleCounter, setAppleCounter] = useState(0);
+    const [kiwiCounter, setKiwiCounter] = useState(0);
+
+    function decreaseCounter(counter, setCounter) {
+        if (counter > 0) {
+            setCounter(counter - 1)
+        }
+    }
+
+    function resetCounters() {
+        setStrawberryCounter(0)
+        setBananaCounter(0)
+        setAppleCounter(0)
+        setKiwiCounter(0)
+    }
+
+    const [formState, setFormState] = useState({
+        firstname: '',
+        lastname: '',
+        age: '0',
+        zipcode: '',
+        deliveryfrequency: 'Iedere week',
+        moment: 'Overdag',
+        message: '',
+        conditions: false
+
+
+    });
+
+    function handleChange(e) {
+        const changedFieldName = e.target.name;
+        const newValue = e.target.type === "checkbox" ?
+            e.target.checked : e.target.value;
+
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue
+        });
+
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault()
+        console.log(strawberryCounter, bananaCounter, appleCounter, kiwiCounter, formState)
+    }
+
+    function convertEmoji(em) {
+        return emoji.getUnicode(em)
+    }
+
+    return (
+        <>
+            <main>
+                <h1>Fruitmand bezorgservice</h1>
+                <Fruitcounter title="Aardbeien"
+                              onClickIncrease={() => setStrawberryCounter(strawberryCounter + 1)}
+                              onClickDecrease={() => decreaseCounter(strawberryCounter, setStrawberryCounter)}
```

Comment:
Als je de decreaseCounter niet nodig hebt, dan kun je het ook wat versimpelen
```suggestion
                              setCounter={setStrawberryCounter}
```
Omdat het component óók de counter al meekrijgt kun je ín het component de functie aanroepen met +1, dan wel -1.

Op deze manier heb je nóg minder code nodig 😉